test/module/shared_model/validators/transaction_validator_test.cpp

```diff
@@ -66,7 +66,11 @@ TEST_F(TransactionValidatorTest, StatelessValidTest) {
   // Iterate through all command types, filling command fields with valid values
   iterateContainer(
       [] { return iroha::protocol::Command::descriptor(); },
-      [&](auto field) {
+      [&](auto field)->google::protobuf::Message* {
+        if (field->full_name() == "iroha.protocol.Command.set_setting_value")
```

Comment:
let's move it to validators fixture and make more general (see my comment in validators_fixture.hpp)

Reply:
Fix it

Reply:
fixed